.changeset/smooth-students-nail copy.md

```diff
@@ -0,0 +1,6 @@
+---
+"@justeattakeaway/pie-button": major
```

Comment:
These will get deleted before merging, just wanted to show the error for review purposes 👍🏼